### PR TITLE
Correct indentation and snake_case namespaces

### PR DESCRIPTION
**Pull request by buschbapti**
It is crazy PR time again. As always this is just an indentation refactor. Also I have changed all the namespaces to adopt snake_case naming convention and remove some of the nested ones.

### CODE REVIEW

File: source/packages/modulo_core/tests/testMovingReferenceFrame.cpp [hunk 1]
```diff
@@ -1,202 +1,173 @@
-#include "modulo_core/Cell.hpp"
-#include "state_representation/Space/Cartesian/CartesianState.hpp"
-#include "state_representation/Space/Cartesian/CartesianPose.hpp"
-#include "state_representation/Space/Cartesian/CartesianTwist.hpp"
 #include "dynamical_systems/Circular.hpp"
```
Comment (domire8) [inline source/packages/modulo_core/tests/testMovingReferenceFrame.cpp:1]: If you want to be very consistent, the state_rep and DS includes should now be treated as external libs (with `<>` and after the local header includes)

File: source/packages/modulo_core/include/modulo_core/Exceptions/NotImplementedExcpetion.hpp [hunk 2]
```diff
@@ -1,23 +1,11 @@
...
-	{
-		class NotImplementedException : public std::logic_error
-		{
-		public:
-			explicit NotImplementedException(const std::string& msg) : logic_error(msg)
-			{};
-		};
-	}
-}
+namespace modulo::core::exceptions {
```
Comment (domire8) [inline source/packages/modulo_core/include/modulo_core/Exceptions/NotImplementedExcpetion.hpp:6]: Getting rid of these nested namespaces is so satisfying :smile:

**Review by domire8 — APPROVED** [review]
Great, the formatting and namespace changes are very nice! I trust that you didn't change the code otherwise and I hope that one day I can use this code for real

File: source/packages/modulo_core/tests/testMovingReferenceFrame.cpp (hunk 1 above)
Comment (buschbapti) [reply source/packages/modulo_core/tests/testMovingReferenceFrame.cpp:1]: that is a good point yes that should be there as well. I will change that

File: source/packages/modulo_core/include/modulo_core/Exceptions/NotImplementedExcpetion.hpp (hunk 2 above)
Comment (buschbapti) [reply source/packages/modulo_core/include/modulo_core/Exceptions/NotImplementedExcpetion.hpp:6]: ah ah yes indeed!

**Pull request merged**